src/Components/ArtworkFilter/ArtworkFilters/__tests__/ArtistSeriesFilter.jest.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  RenderOptions,
+  render as originalRender,
+  screen,
+} from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import {
+  ArtworkFilterContextProps,
+  ArtworkFilterContextProvider,
+  useArtworkFilterContext,
+} from "Components/ArtworkFilter/ArtworkFilterContext"
+import { ArtistSeriesFilter } from "Components/ArtworkFilter/ArtworkFilters/ArtistSeriesFilter"
+import { ReactElement } from "react"
+
+const render = (ui: ReactElement, options: RenderOptions = {}) =>
+  originalRender(ui, { wrapper: Wrapper, ...options })
+
+const Wrapper: React.FC = ({ children }) => {
+  return (
+    <ArtworkFilterContextProvider
+      aggregations={[
+        {
+          slice: "ARTIST_SERIES",
+          counts: [
+            { name: "Companions", value: "kaws-companions", count: 42 },
+            { name: "Toys", value: "kaws-toys", count: 420 },
+          ],
+        },
+      ]}
+    >
+      <ClearAllButton />
+      {children}
+      <ArtworkFilterContextInspector />
```

Comment:
Much of the boilerplate in this file for asserting about the filter state was introduced in the original react-testing-library [spike](https://github.com/artsy/force/pull/7629#discussion_r639742956), and has been used in the couple of filters that were created since then. 

This is the third such duplication, so maybe it's time to extract, but I'll leave that for a separate refactor PR.

Reply:
Split this off to https://github.com/artsy/force/pull/13052, and will rebase here once that's merged

Reply:
Rebased and updated in https://github.com/artsy/force/pull/13041/commits/63d4c54b72f83431fcbac5d43366ffc88957db6d